src/main/java/org/opentripplanner/middleware/utils/GraphQLUtils.java

```diff
@@ -0,0 +1,51 @@
+package org.opentripplanner.middleware.utils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.*;
+
+public class GraphQLUtils {
+    private static final Logger LOG = LoggerFactory.getLogger(GraphQLUtils.class);
+
+    // Lazily-initialized in getPlanQueryTemplate()
+    private static String planQueryTemplate = null;
+
+    /**
+     * Location of the GraphQL plan query template file, as Java resource.
+     */
+    public static final String PLAN_QUERY_RESOURCE = "queries/planQuery.graphql";
+
+    /**
+     * Return the full GraphQL plan file planQueryTemplate in Java string format, with {@code "} as {@code \"}
+     */
+    public static String getPlanQueryTemplate() {
```

Comment:
This method does not appear to be called which suggests that the plan query is not loaded. This class also isn't referenced anywhere that I can see, is it needed?

Reply:
You may be right, the bulk of this PR deals with the result of the call to the GraphQL endpoint, not the call itself. Totally missed that. In that case, should I remove the code that breaks the itinerary checks?